src/test/java/com/puppycrawl/tools/checkstyle/internal/XpathRegressionTest.java

```diff
@@ -114,7 +170,13 @@
 
     @BeforeAll
     public static void setUpBeforeClass() throws IOException {
-        simpleCheckNames = CheckUtil.getSimpleNames(CheckUtil.getCheckstyleChecks());
+        final Set<Class<?>> checks = CheckUtil.getCheckstyleChecks()
+                .stream()
+                // Javadoc checks are not compatible with SuppressionXpathFilter
+                // till https://github.com/checkstyle/checkstyle/issues/5770
+                .filter(check -> !AbstractJavadocCheck.class.isAssignableFrom(check))
```

Comment:
Again it will take more than just this 1 issue. We would specifically need to build xpath support for DetailNode. See https://github.com/checkstyle/checkstyle/tree/master/src/main/java/com/puppycrawl/tools/checkstyle/xpath

Reply:
Once this issue is resolved, all `AbstractJavadocCheck` should be added to the list of incompatible checks. If we already have such an issue for this, please specify.

Reply:
As far as I know, we do not have an issue on it as we are still trying to finish up DetailAST support in xpath.

Reply:
I agree that another issue will be needed. Perhaps even a number of them. But I'm not ready to start such issue yet.

Reply:
If this line won't be removed with that issue number, then I would just not reference an issue right now. If you expect the line to be removed with the issue, then make a remark on the issue so it isn't forgotten.

Reply:
done